packages/webapp/src/components/Form/Unit/index.jsx

```diff
@@ -115,12 +115,12 @@ const useReactSelectStyles = (disabled, { reactSelectWidth = DEFAULT_REACT_SELEC
         width: `${reactSelectWidth - 19}px`,
         display: 'flex',
         justifyContent: 'center',
-        background: state.isDisabled ? 'var(--inputDisabled)' : 'inherit',
+        background: disabled ? 'var(--inputDisabled)' : 'inherit',
```

Comment:
Do not use disabled outside of reactSelect here. 
disabled is passed to the react select component, and then state.isDisabled is synced internally in reactSelect library.
state.isDisabled should always be up to date.

Reply:
@JimmyRowland 

The reason I changed this is because I need the Unit based disabled distinction to render this css. Essentially: disabled != isDisabled. For Select we set the attribute as: 
`isDisabled = {isSelectDisabled || disabled} ` for units. 

Since disabled is about Unit and not about Select I think the pattern here is fine until the css pattern is updated as a whole, since it is passed to Select through Unit and the useMemo handles the update.

Let me know if you have another pattern in mind for this distinction.

Reply:
Thanks. Then isSelectDisabled || disabled is definitely wrong.  || disabled should go into useMemo

Reply:
The component has completely changed since the last time I looked it. Ignore my comments if they do not make sense.

Reply:
@JimmyRowland 

Thanks. It was a good thing to double check its a weird component I second guessed it too. It made sense. I don't ignore though haha we always need more approvers/reviewers!

Reply:
I just tried adding disable to useMemo and it caused problems. Double checking the react docs I am seeing a note that says [The array of dependencies is not passed as arguments to the function.](https://reactjs.org/docs/hooks-reference.html#usememo) .. just the dependency array like useEffect.

Reply:
https://beta.reactjs.org/apis/react/useMemo#examples-recalculation

Reply:
Something might be seriously wrong with unit component